mne_bids_pipeline/_config.py

```diff
@@ -2089,22 +2089,31 @@ def noise_cov(bids_path):
 of `mne.compute_covariance` for details.
 """
 
-source_info_path_update: dict[str, str] | None = dict(suffix="ave")
+source_info_path_update: dict[str, str] | None = None
```

Comment:
Yikes, looks like `None` was documented as an option but never actually supported based on the code changes 🤭. Lines like `bids_path.copy().update(**cfg.source_info_path_update)` would have just died...

In any case, now that `None` is the default we can also remove this line:

https://github.com/mne-tools/mne-bids-pipeline/blob/d8c1e4b683f45567f0c1aad59ab87c89355bf34b/mne_bids_pipeline/_config_import.py#L347

Reply:
done